noir-projects/noir-contracts/contracts/fpc_contract/src/lib.nr

```diff
@@ -1,10 +0,0 @@
-use dep::aztec::context::PublicContext;
-
-pub fn compute_rebate(context: PublicContext, initial_amount: Field) -> Field {
```

Comment:
This function was invoked only in one place so having this separate just cluttered it.